docs/dev/agent_api.md

```diff
@@ -1,10 +1,16 @@
 # IPC (Inter Process Communication) API
 
 The agent communicates with the outside world through an HTTP API to ease the
-development of 3rd party tools and interfaces. Since HTTP is transported over
-a [Unix Socket](https://en.wikipedia.org/wiki/Unix_domain_socket) on *nix platforms
-and [Named Pipes](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365590.aspx)
-on Windows, authorization is delegated to the filesystem.
+development of 3rd party tools and interfaces. By default, the API is available
+at TCP port `5001` on `localhost` but can be configured differently.
+
+## Security and Authentication
+
+To avoid unprivileged users to access the API, authentication is required.
+
+FIXME: new authentication system wasn't merged yet: https://github.com/DataDog/datadog-agent/pull/939
```

Comment:
fyi it's merged now, will be in `beta.7`

Reply:
Merging for now, that would be a fair large addition to the docs